Dockerfile.processing_image

```diff
@@ -21,7 +21,7 @@ RUN git clone https://github.com/chanzuckerberg/single-cell-explorer.git \
 # Install python dependencies
 # Remove packaging dependency once pyparser>3 is supported.
 RUN pip3 install scanpy==1.6.0 python-igraph==0.8.3 louvain==0.7.0 packaging==21.0 awscli
-RUN pip3 install cellxgene-schema==2.0.3
+RUN pip3 install cellxgene-schema==2.0.5
```

Comment:
Note: This will fail until this PR is merged and released to PyPI:
* https://github.com/chanzuckerberg/single-cell-curation/pull/172